chall01/dmukaliy.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+import sys
+
+def error():
```

Comment:
it's really DRY concept to create a function error as you know you will use it many times. I would just maybe recommend using another name than: error() which is too generic by something like `print_usage()` or `print_and_exit(string)`